<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDialog</class>
 <widget class="QDialog" name="DataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="optionsHorizontalLayout">
       <item>
        <widget class="QCheckBox" name="oneOriginCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1-origin data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendToConsolePushButton">
         <property name="minimumSize">
          <size>
           <width>118</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Send to console</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="consoleOptionsPushButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="consoleDataOptionsGroupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="title">
        <string>Console data options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="variableNameHorizontalLayout">
          <item>
           <widget class="QLabel" name="variableNameLabel">
            <property name="text">
             <string>Enter name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="variableNameLineEdit">
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="moveDataRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move to console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="copyDataRadioButton">
          <property name="text">
           <string>Copy to console</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>oneOriginCheckBox</tabstop>
  <tabstop>consoleOptionsPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>consoleOptionsPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>consoleDataOptionsGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
